src/client/src/main.tsx

```diff
@@ -43,21 +45,22 @@ const appBootstrapper = () => {
 
   const applicationDependencies = createApplicationServices(user, autobahn, openFin)
 
-  const store = configureStore(applicationDependencies)
-  window.store = store
+  const store = (window.store = configureStore(applicationDependencies))
 
   ReactDOM.render(
     <Provider store={store}>
       <EnvironmentProvider value={environmentContext}>
-        <ThemeProvider>
-          {openFin.isRunningInOpenFin ? (
-            <OpenFinProvider openFin={openFin}>
+        <ThemeState.Provider name="light">
+          <DeprecatedUITheme.ThemeProvider>
```

Comment:
Is this really deprecated? I see how you this PR lets you change themes but it does not change the way we consume themes.